docs/start/getting-started/fragments/android/native_integrate.md

```diff
@@ -90,7 +90,7 @@ Next, you'll create a Todo and save it to DataStore.
   ```java
     Amplify.DataStore.save(
             item,
-            success -> Log.i("Tutorial", "Saved item: " + success.item.getName()),
+            success -> Log.i("Tutorial", "Saved item: " + success.item().getName()),
```

Comment:
```suggestion
            success -> Log.i("Tutorial", "Saved item: " + item.getName()),
```

Reply:
I think it's better to keep referencing `success.item()`.  Theoretically it should be the same as `item`, but good to confirm.